lib/ansible/module_utils/k8s/raw.py

```diff
@@ -210,18 +210,19 @@ def perform_action(self, resource, definition):
             if self.check_mode:
                 k8s_obj = dict_merge(existing.to_dict(), definition)
             else:
-                if self.params['merge_type']:
-                    from distutils.version import LooseVersion
-                    if LooseVersion(self.openshift_version) < LooseVersion("0.6.2"):
+                from distutils.version import LooseVersion
```

Comment:
Why isn't this being imported at the top of the file?

Reply:
It's only needed for this one thing so makes sense to only import on demand? But if that doesn't make sense then it can be moved.